src/test/java/seedu/planitarium/storage/StorageTest.java

```diff
@@ -102,4 +92,11 @@ void loadData_invalidQuery_expectAssertionError() {
         }
     }
 
+    @AfterAll
+    static void wipeData() {
+        familyTest = new Family();
+        storageSave = new Storage();
+        storageSave.saveData(familyTest);
```

Comment:
Is this line necessary? `familyTest` would be newly initialised and have nothing to store right?